doc/source/development/contributing_codebase.rst

```diff
@@ -807,8 +804,45 @@ across the number of specified cores or ``auto`` to utilize all the available co
    # Utilizes all available cores
    pytest -n auto pandas
 
-This can significantly reduce the time it takes to locally run tests before
-submitting a pull request.
+If you'd like to speed things along further a more advanced use of this
+command would look like this
+
+.. code-block:: bash
+
+    pytest pandas --skip-slow --skip-network --skip-db -m "not single_cpu" -n 4 -r sxX
+
+In addition to the multithreaded performance increase this improves test
+speed by skipping some tests:
+
+- skip-slow: any test that takes long (think seconds rather than milliseconds)
+- skip-network: tests that require network connectivity
+- skip-db: tests that require a database
+
+The -r flag will display a short summary info (see `pytest documentation <https://docs.pytest.org/en/4.6.x/usage.html#detailed-summary-report>`_)
+. Here we are displaying the number of:
+
+- s: skipped tests
+- x: xfailed tests
+- X: xpassed tests
+
+This is optional and can be removed if you don't like the summary. Using this
```

Comment:
Maybe ``Using the parallelization option``. This sounds a bit like its referencing the -r

Reply:
I meant the summary because in the original question some folks mentioned they didn't like that it was included in the script. I want to clarify that adding it to the “advanced command” isn't helpful for speed, it only adds the summary. I can clarify this.